src/components/TodoContent/TodoContent.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { ChangeEvent, FormEvent, useEffect, useState } from 'react';
+import { usePostTodos } from '../../hooks/usePostTodos';
+import { useTodos } from '../../utils/TodoContext';
+
+export const TodoContent: React.FC<{ children: React.ReactNode }> = ({
+  children,
+}) => {
+  const [input, setInput] = useState<string>('');
```

Comment:
don't need type cos it's the same as initial value (find and fix all instances where it's not needed)